client/src/App.js

```diff
@@ -0,0 +1,125 @@
+import './App.css';
```

Comment:
I think the general structure you have here of "App", "Question", "Flag" is pretty good!